src/control/cmd/dmg/pool_test.go

```diff
@@ -614,13 +614,14 @@ func TestPoolCommands(t *testing.T) {
 			}, " "),
 			nil,
 		},
+		// Exclude testing with multiple ranks is verified at the control API layer.
```

Comment:
Does it hurt anything to verify it here as well? There seems to be value to me in ensuring rank lists are parsed correctly at this level.

Reply:
currently this would require significant reworking of the command_test.go code because we have multiple layers of Control API calls now. I don't want to do that as part of this PR

Reply:
Oh, I didn't realize. Yeah, if it requires some refactoring, let's leave it for a future improvement.